src/django_components/expression.py

```diff
@@ -203,14 +138,8 @@ def is_dynamic_expression(value: Any) -> bool:
     return True
 
 
-def is_spread_operator(value: Any) -> bool:
-    if not isinstance(value, str) or not value:
-        return False
-
-    return value.startswith("...")
-
-
-def process_aggregate_kwargs(kwargs: Mapping[str, Any]) -> Dict[str, Any]:
+# TODO - Move this out into a plugin?
+def process_aggregate_kwargs(params: List["TagParam"]) -> List["TagParam"]:
```

Comment:
Interestingly, with the new flow, the `attr:key=val` syntax could be later redefined as an (internal) plugin.